upup/models/cloudup/resources/addons/networking.projectcalico.org.canal/k8s-1.16.yaml.template

```diff
@@ -522,6 +529,10 @@ spec:
         k8s-app: calico-typha
         role.kubernetes.io/networking: "1"
       annotations:
+        # This, along with the CriticalAddonsOnly toleration below, marks the pod as a critical
+        # add-on, ensuring it gets priority scheduling and that its resources are reserved
+        # if it ever gets evicted.
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
This annotation is removed from 1.16 onwards: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md#deprecations-and-removals